org.prismlauncher.PrismLauncher.yml

```
@@ -58,8 +60,31 @@ modules:
       - mv /app/jre /app/jdk/17
       - /usr/lib/sdk/openjdk8/install.sh
       - mv /app/jre /app/jdk/8
+    cleanup: [/jre]
```

Nit: This change reverts a styling change I introduced in 675be8f8f193c6e413168f92e51de60317ca61b7, so for consistency with other modules, I'd keep YAML arrays as lists, i.e.:

```suggestion
    cleanup:
      - /jre
```